### ISSUE TITLE
Reference to disassociated pointer, model%inversion%powerlaw_c_inversion

### ISSUE THREAD

**Issue by billsacks** [ESCOMP/cism#2]
Tests with nag in debug mode die with this error:

```
/home/sacks/cesm_code/cism-wrapper/source_cism/libglissade/glissade_velo_higher.F90,Reference to disassociated POINTER MODEL%INVERSION%POWERLAW_C_INVERSION
```

pointing to this line:

https://github.com/ESCOMP/cism/blob/a2c7ce0c2a7aa49909252ed82ba2b2920604e700/libglissade/glissade_velo_higher.F90#L1075

Example test: `SMS_D_Ly1_P24x1.f09_g17_gl4.T1850G.hobart_nag`

**Comment by billsacks** [comment]
This death is in model initialization.

**Comment by whlipscomb** [comment]
@billsacks, Please try the fix in the new branch lipscomb/bug_fixes

**Comment by billsacks** [comment]
Fixed by ff3091c6
